backend/tests/facilities_test.go

```diff
@@ -76,7 +76,7 @@ func TestCreateFacility(t *testing.T) {
 		}
 		created := facilities.Data[0]
 		if strings.Compare(created.Name, "TestingFacility") != 0 {
-			t.Errorf("incorrect output, expected: TestingFacility, got: " + created.Name)
+			t.Errorf("incorrect output, expected: TestingFacility, got %v", created.Name)
```

Comment:
The pre-commit hook kept failing because the Go linter started flagging the original line of code. Rather than `--no-verify` I addressed the line that caused it to fail.